src/components/PersonTableRow.tsx

```diff
@@ -0,0 +1,53 @@
+import { v4 as uuid } from 'uuid';
+import cn from 'classnames';
+
+import { NO_PARENT } from '../constans/personConstants';
+
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  selected: boolean;
+};
+
+export const PeopleTableRow: React.FC<Props> = ({
+  person,
+  people,
+  selected,
+}) => {
+  const { sex, born, died, motherName, fatherName } = person;
+
+  const mother = people.find(mom => mom.name === motherName);
+  const father = people.find(dad => dad.name === fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      key={uuid()}
```

Comment:
Using `uuid()` for keys in a list can lead to unnecessary re-renders because a new UUID is generated on each render. Consider using a stable key, such as `person.slug` or another unique identifier from the `person` object, to ensure consistent keys across renders.